docs/commands/mftf.md

```diff
@@ -468,14 +468,20 @@ vendor/bin/mftf static-checks testDependencies
 vendor/bin/mftf static-checks actionGroupArguments
 ```
 ```bash
+vendor/bin/mftf static-checks deprecatedEntityUsage
```

Comment:
Let's also add the --path option for this static check

Reply:
done

Reply:
rectified. Since these are examples, added 2 usages, one with and one without `-p`